src/Administration/Resources/app/administration/src/module/sw-cms/component/sw-cms-slot/sw-cms-slot.scss

```diff
@@ -124,15 +124,29 @@
             margin: auto;
             position: absolute;
             top: 0;
-            left: 0;
-            right: 0;
             bottom: 0;
             background: rgba(255, 255, 255, 80%);
             border-radius: 5px;
+
+            &.element-selection__overlay-action-select {
+                left: 0;
+            }
+
+            &.element-selection__overlay-action-favorite {
+                right: 0;
+            }
         }
     }
 }
 
+.sw-cms-slot__modal-container {
```

Comment:
Again, please scope <3

Reply:
Unsure whether I can add something useful. Shall I prepend `.sw-modal`?

<img width="410" alt="image" src="https://user-images.githubusercontent.com/1133593/180645396-a0502cd8-158d-44a5-b5b6-9a26e9952746.png">

Reply:
Well, I see, thats hierachically not in sw-cms-slot, but since it's named like that, you should at least put it into ".sw-cms-slot" above, even if you just use `&__model-container` after words, for structure's sake